src/main/java/com/simprints/libsimprints/Registration.kt

```diff
@@ -0,0 +1,35 @@
+package com.simprints.libsimprints;
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@SuppressWarnings("unused", "WeakerAccess")
+@Parcelize
+data class Registration @JvmOverloads constructor(
+    val guid: String,
+    private val templates: MutableMap<FingerIdentifier, ByteArray> = mutableMapOf()
+) : Parcelable {
+
+    fun setTemplate(fingerId: FingerIdentifier, fingerTemplate: ByteArray) {
```

Comment:
What about face temeplates?

Reply:
Hmm, I just translated the code explicitly not changing the API.

I will add a subtask to add the face templates well for the "improvements" part